components/k8s-model-server/README.md

```diff
@@ -163,3 +163,148 @@ You can learn more about [updating a Deployment](https://kubernetes.io/docs/conc
 [scaling a Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment), and 
 [Pod Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) in the 
 Kubernetes documentation.
+
+
+
+### Use the served model
+
+The [inception-client](./inception-client) directory contains a Python script you can use to make a call against the deployed model.
+
+This script is intended to be run externally to the kubernetes cluster as a demonstration that the inception model is correctly being served.
+You can run the script either directly from a Python2 environment or in a Docker container.
+
+#### Setup
+
+You will require the external IP for the inception service as well as the port it is being hosted on. The inception service should be
+listed under the value you used for the `MODEL_NAME` parameter in the ksonnet component. You can find this information using
+```commandline
+kubectl get services
+NAME         TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)			 AGE
+$MODEL_NAME  LoadBalancer   <INTERNAL IP>   <SERVICE IP>     <SERVICE PORT>:<NODE PORT>  <TIME SINCE DEPLOYMENT>
+```
+
+We will feed the `<SERVICE IP>` and `<SERVICE PORT>` to the labelling script. We will use it to label the following image of a
+cat sleeping on a comforter atop a sofa:
+
+![Cat on comforter on sofa](./inception-client/sleeping-pepper.jpg)
+
+You can also use to to label your own images.
+
+#### Running the script directly
+
+You can run the script directly in your local environment if Python2 is available to you. You will not be able to use the script with Python3
+as the [`tensorflow-serving-api` package](https://pypi.python.org/pypi/tensorflow-serving-api)
+is not yet Python3-capable ([Issue #117](https://github.com/google/kubeflow/issues/117)).
+
+If you would like to use a virtual environment, begin by activating your desired environment with your favorite environment manager. Then,
+```commandline
+pip install -r requirements.txt
+```
+
+Run the script as follows:
+
+```commandline
+python label.py -s <SERVICE IP> -p <SERVICE PORT> sleeping-pepper.jpg
+```
+
+#### Run in Docker container with publicly exposed service
+
+The [inception-client](./inception-client) directory also contains a [Dockerfile](./inception-client/Dockerfile) that will allow you to
+call out to the inception service from a container. You can run this container on your local machine if you publicly exposed your
+`inception` service. If you would like to do this on GKE, simply run
+
+```commandline
+kubectl edit service inception
+```
+
+and change the service type to `NodePort` or `LoadBalancer`.
```

Comment:
I'd suggest removing NodePort option here.

If they use NodePort they'd have to make sure firewall rules don't block access to the nodes; they'd also have to get IP of their nodes.